docs/apache-airflow-providers-amazon/operators/bedrock.rst

```diff
@@ -174,6 +177,40 @@ To add data from an Amazon S3 bucket into an Amazon Bedrock Data Source, you can
     :start-after: [START howto_operator_bedrock_ingest_data]
     :end-before: [END howto_operator_bedrock_ingest_data]
 
+.. _howto/operator:BedrockRetrieveOperator:
+
+Amazon Bedrock Retrieve
+=======================
+
+To query a knowledge base, you can use :class:`~airflow.providers.amazon.aws.operators.bedrock.BedrockRaGOperator`.
```

Comment:
This should be the `BedrockRetrieveOperator`, right? Or maybe the one below? Right now RAG is in both. I also don't fully understand the difference between the two operators. Maybe just a sentence or two more to tell the two apart? The only difference I see in the description is one says "includes" the other says "cites"

Reply:
> This should be the BedrockRetrieveOperator, right?

You are right

> I also don't fully understand the difference

I'll add some more context tot he docs and see if that helps.